bundle/bundle.go

```diff
@@ -151,6 +153,21 @@ func (b *Bundle) CacheDir(paths ...string) (string, error) {
 	return dir, nil
 }
 
+func (b *Bundle) InternalDir() (string, error) {
+	cacheDir, err := b.CacheDir()
+	if err != nil {
+		return "", err
+	}
+
+	dir := filepath.Join(cacheDir, InternalFolder)
+	err = os.MkdirAll(dir, 0700)
+	if err != nil {
+		return dir, err
+	}
+
+	return dir, nil
+}
```

Comment:
Please add a comment what this directory should be used for, and if it's always ephemeral or not (it is by nature of being a subdir of the cache dir, but that won't be clear from the call sites).